OnePlus/N200/res/xml/power_profile.xml

```diff
@@ -0,0 +1,109 @@
+    <item name="dsp.audio">16</item>
+    <item name="dsp.video">41</item>
+    <item name="bluetooth.controller.idle">1.7</item>
+    <item name="bluetooth.controller.rx">76</item>
+    <item name="bluetooth.controller.tx">176</item>
+    <item name="bluetooth.controller.voltage">3300</item>
+    <item name="radio.active">191.5</item>
+    <item name="radio.scanning">143.5</item>
+    <array name="radio.on">
+        <value>1</value>
+    </array>
+    <item name="modem.controller.idle">1</item>
+    <item name="modem.controller.rx">200.8</item>
+    <array name="modem.controller.tx">
+        <value>176.4</value>
+        <value>246.8</value>
+        <value>323.3</value>
+        <value>484.6</value>
+        <value>646.8</value>
+    </array>
+    <item name="modem.controller.voltage">800</item>
+    <array name="cpu.clusters.cores">
+        <value>4</value>
+        <value>2</value>
+    </array>
+    <array name="cpu.core_speeds.cluster0">
+        <value>300000</value>
+        <value>576000 </value>
+        <value>710400 </value>
+        <value>940800 </value>
+        <value>1190400 </value>
+        <value>1324800 </value>
+        <value>1478400 </value>
+        <value>1574400 </value>
+        <value>1708800 </value>
+        <value>1804800 </value>
+    </array>
+    <array name="cpu.core_speeds.cluster1">
+        <value>768000 </value>
+        <value>979200 </value>
+        <value>1017600 </value>
+        <value>1248000 </value>
+        <value>1401600 </value>
+        <value>1536000 </value>
+        <value>1651200 </value>
+        <value>1804800 </value>
+        <value>2035200 </value>
+    </array>
+    <array name="cpu.core_power.cluster0">
+        <value>30</value>
+        <value>32</value>
+        <value>34</value>
+        <value>40</value>
+        <value>47</value>
+        <value>56</value>
+        <value>60</value>
+        <value>62</value>
+        <value>67</value>
+        <value>72</value>
+    </array>
+    <array name="cpu.core_power.cluster1">
+        <value>62</value>
+        <value>79</value>
+        <value>87</value>
+        <value>100</value>
+        <value>114</value>
+        <value>136</value>
+        <value>151</value>
+        <value>180</value>
+        <value>234</value>
+    </array>
+    <item name="cpu.awake">15.68</item>
+    <item name="cpu.idle">6.5</item>
+    <array name="memory.bandwidths">
+        <value>22.7</value>
+    </array>
+    <item name="battery.capacity">5000</item>
+    <item name="wifi.controller.idle">1.4</item>
+    <item name="wifi.controller.rx">177</item>
+    <item name="wifi.controller.tx">269</item>
+    <array name="wifi.controller.tx_levels" />
+    <item name="wifi.controller.voltage">3300</item>
+    <array name="wifi.batchedscan">
+        <value>.0002</value>
+        <value>.002</value>
+        <value>.02</value>
+        <value>.2</value>
+        <value>2</value>
+    </array>
+    <allow-in-power-save package="com.google.android.gms" />

```

Comment:
not that i care, but I'm pretty sure this shouldn't be there

Reply:
Taken from stock as-is ofc